linode/manager
# feat: [M3-7267] - Add AGLB Rule Delete Dialog

## Description 📝

- Adds the functionality to delete rules from an AGLB route 🗑️🎉

## Preview 📷

![Screenshot 2023-10-17 at 11 51 56 AM](https://github.com/linode/manager/assets/115251059/f4d75a64-22bd-4552-9295-be9ad48671ea)

## How to test 🧪

### Prerequisites
- Turn on the MSW
- Navigate to `http://localhost:3000/loadbalancers/0/routes`

### Verification steps 
- Expand any of the Routes ⬇️
- Attempt to delete a rule on a route 🗑️
- Verify the route gets deleted ☑️

Test:
```
yarn cy:run -s "cypress/e2e/core/loadBalancers/load-balancer-routes.spec.ts"
```

## As an Author I have considered 🤔

*Check all that apply*
